app/src/pages/common/modals/importModal/importModal.jsx

```diff
@@ -222,6 +224,14 @@ export class ImportModal extends Component {
   successUploadHandler = (id) => {
     this.props.data.onImport();
     const { files } = this.state;
+    const {
+      tracking: { trackEvent },
+      data: {
+        eventsInfo: { uploadButton },
+      },
+    } = this.props;
+
+    trackEvent(uploadButton(this.getFilesNames(files)));
```

Comment:
I suppose you need to add additional checks for such props availability, as they are part of the `data` object, their default values won't work in case the object will be passed to the component.